tasks/config_kvm.yml

```diff
@@ -37,4 +37,8 @@
   with_items:
     - '/etc/apparmor.d/usr.sbin.libvirtd'
     - '/etc/apparmor.d/usr.lib.libvirt.virt-aa-helper'
-  when: libvirt_apparmor_disabled.changed
+  when: >
+        libvirt_apparmor_disabled.changed and
+        enable_apparmor is defined and
```

Comment:
We should define `enable_apparmor` in `defaults/main.yml`. But also let's change it to:
```
kvm_enable_apparmor: true
```